staging/istio/Chart.yaml

```diff
@@ -1,21 +1,19 @@
 apiVersion: v2
 name: istio
-version: 1.16.2
-appVersion: 1.16.2
-description: Helm chart for Istio
+version: 1.16.3
+appVersion: 1.16.3
```

Comment:
I bumped the patch version here to get CI passing. will follow up with upgrade to 1.17.3 upstream after this merges